build.gradle.kts

```diff
@@ -1,36 +1,45 @@
 import com.github.jk1.license.LicenseReportExtension
-import io.spring.gradle.bintray.SpringBintrayExtension
+//import io.spring.gradle.bintray.SpringBintrayExtension
 import nebula.plugin.contacts.Contact
 import nebula.plugin.contacts.ContactsExtension
 import nebula.plugin.info.InfoBrokerPlugin
 import nl.javadude.gradle.plugins.license.LicenseExtension
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
-import org.jfrog.gradle.plugin.artifactory.dsl.ArtifactoryPluginConvention
-import org.jfrog.gradle.plugin.artifactory.dsl.PublisherConfig
+//import org.jfrog.gradle.plugin.artifactory.dsl.ArtifactoryPluginConvention
```

Comment:
Could we just go ahead and remove these?

Reply:
yes definitely, I'm just committing this as I work through some of the compilation errors in subprojects